test/Directory.Build.props

```diff
@@ -31,6 +31,19 @@
     <PackageReference Include="xunit" Version="2.4.0" />
     <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
     <PackageReference Include="Microsoft.DotNet.XUnitExtensions" Version="2.4.0-prerelease-63213-02" />
+    <PackageReference Include="coverlet.msbuild" Version="$(CoverletVersion)" />
```

Comment:
@ViktorHofer is this dependency OK in all test projects?

Reply:
jep. the msbuild package is used for that reason. It adds targets that can be used on a project level: https://github.com/tonerdo/coverlet/blob/master/src/coverlet.msbuild/coverlet.msbuild.targets